src/StudyEntities/Candidate/ValidatableIdentifier.php

```diff
@@ -55,12 +55,11 @@ abstract class ValidatableIdentifier implements Identifier
      *
      * @throws \DomainException When the value is not valid
      */
-    public final function __construct(string $value)
+    final public function __construct(string $value)
```

Comment:
Why is this declared final? What if a subclass needs to change it?

Reply:
IIRC it's so that every other identifier is forced to call its validate function.

Reply:
Whether it's a good idea or not, I don't see why enforcing PSR2 would mandate that, though.. it seems like more than a stylistic change.

Reply:
It was final before -- this just changes the order of the keywords.

Reply:
Oh, nevermind then. GitHub is only highlighting one of the words, so I didn't catch that.